docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+    :language: xml
+    :start-after: <!-->XML-HISTORY<-->
+    :end-before: <!--><-->
+
+.. _latencybudgetqospolicy:
+
+LatencyBudgetQosPolicy
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    This QoS Policy will be implemented in future releases.
+
+This QoS Policy urgency of the data communication, specifying the maximum acceptable delay from the time the data is
+written until the data is inserted on the cache of the receiver application and notified of the fact.
+That delay by default is set to 0 in order to optimize the internal operations.
+
+List of QoS Policy data members:
+
++--------------------------+-------------------------------+-------------------+
+| Data Member Name         | Type                          | Default Value     |
++==========================+===============================+===================+
+| duration                 | fastrtps::Duration_t          | c_TimeZero        |
++--------------------------+-------------------------------+-------------------+
+
+.. note::
+     This QoS Policy concerns to Topic, DataWriter and DataReader entities.
+     :raw-html:`<br />`
+     It can be changed after the creation of the entities.
+
+.. warning::
+    For DataWriters and DataReaders to match, they must follow the compatibility rule. See :ref:`latencybudget_compatibilityrule`
+    for further details.
+
+.. _latencybudget_compatibilityrule:
+
+Compatibility Rule
+""""""""""""""""""
+
+To maintain the compatibility between LatencyBudgetQosPolicy in DataReaders and DataWriters, the DataWriter duration
+must be lower or equal to the DataReader duration.
+
+.. _lifespanqospolicy:
+
+LifespanQosPolicy
+^^^^^^^^^^^^^^^^^
+
+Each data sample written by a DataWriter has an associated expiration time beyond which the data is removed from the
+DataWriter and DataReader history as well as from the transient and persistent information caches.
+
+By default, the `duration` is infinite, which means that there isn't a maximum duration for the validity of the samples
+written by the DataWriter.
+
+The expiration time is computed by adding the `duration` to the source timestamp, which can be calculated automatically
+if :func:`write` member function is called or supplied by the application by means of :func:`write_w_timestamp` member
+function.
+The DataReader is allowed to use the reception timestamp instead of the source timestamp.
+
+
+List of QoS Policy data members:
+
++--------------------------+-------------------------------+-------------------+
+| Data Member Name         | Type                          | Default Value     |
++==========================+===============================+===================+
+| duration                 | fastrtps::Duration_t          | c_TimeInfinite    |
++--------------------------+-------------------------------+-------------------+
+
+.. note::
+     This QoS Policy concerns to Topic and DataWriter entities.
+     :raw-html:`<br />`
+     It can be changed after the creation of the entities.
```

Comment:
Although it is an extension, we also have lifespan on DataReader. In fact, the XML example is configuring lifespan on the DataReader/Subscriber